src/main/java/com/revlearn/team1/Team1Application.java

```diff
@@ -1,9 +1,27 @@
 package com.revlearn.team1;
 
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.info.Contact;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.info.License;
+import io.swagger.v3.oas.annotations.servers.Server;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
+@OpenAPIDefinition(
```

Comment:
Can you create a separate file in the config folder named OpenAPIConfig.java or something like that and put this here with the @Configuration bean? Let's keep separation of concerns so root should only be for starting up the application